### ISSUE TITLE
Add ability to limit the number of items of <post_type> to export

### ISSUE THREAD

**Issue by drzraf** [wp-cli/export-command#14]
`--limit=<int>` would be useful to generate a limited set of fixtures from live data.

**Comment by danielbachhuber** [comment]
👍

**Comment by drzraf** [comment]
```
<?xml version="1.0"?>
<!--
    Copyright (C) 2017, Raphaël . Droz + floss @ gmail DOT com
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    Apply transformation of WordPress export files (XHR XML dump).

    $ xsltproc  -stringparam type article -stringparam limit 15  wp-dump-limit.xsl <file.xml>

    These will:
    * strip node from a type != <type>
    * limit the dump to only the first <limit> post (of type <type>)
    Note that <xsl:template match=""> does not accept xsl:parameter
-->
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:wp="http://wordpress.org/export/1.2/" version="1.0">
  <xsl:param name="limit" select="15" />
  <xsl:param name="type" select="page" />
  <xsl:template match="@* | node()">
    <xsl:if test='name() != "item" or wp:post_type != $type or count(preceding-sibling::item[wp:post_type = $type]) &lt; $limit'>
      <xsl:copy>
	<xsl:apply-templates select="@* | node()"/>
      </xsl:copy>
    </xsl:if>
  </xsl:template>
</xsl:stylesheet>
```

(sorry but I can't afford the time required to get "good enough" PR merged inside wp-cli)

**Comment by drzraf** [comment]
pr #15 btw

**Comment by schlessera** [comment]
> (sorry but I can't afford the time required to get "good enough" PR merged inside wp-cli)

Feel free to come chat with us in the [#cli](https://wordpress.slack.com/messages/C02RP4T41) channel if you have trouble with the process or are not sure how to best go about doing something.

**Comment by schlessera** [comment]
Fixed with #15
